### PR TITLE
Adding placeholder text and a link to access urls

### LINKED ISSUE #2785: change the access URL to include all options
### Contributions

- [X] I have read the contribution docs, and understand what's expected of me.

### Link to the page on docs.getdbt.com requiring updates

Found on many pages, such as https://docs.getdbt.com/docs/dbt-cloud-apis/metadata-api

### What part(s) of the page would you like to see updated?

We need to replace cloud.getdbt.com with placeholder text so people who use a different access URL are no longer confused and report cases to support.
- [ ] Search for cloud.getdbt.com and replace with placeholder text and a link to the https://docs.getdbt.com/docs/deploy/regions-ip-addresses page
- [ ] Fix any links broken by the change

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by runleonarun**
closes https://github.com/dbt-labs/docs.getdbt.com/issues/2785

## What are you changing in this pull request and why?

People can be confused when we use cloud.dbt.com in examples when they have different access URLs, so we're adding placeholder text and a link to the Regions & IP Addresses doc. This is [called out in our style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md#placeholder-text) as the appropriate way to deal with this. 

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/dbt-cloud-apis/metadata-querying.md [hunk 1]
```diff
@@ -6,33 +6,36 @@ Accounts on the _Team_ and _Multi-Tenant Enterprise_ plans can query the dbt Met
...
 
 There are several illustrative example queries in this documentation (examples of queries on the Model node, [here](/docs/dbt-cloud-apis/metadata-schema-model).
 
 ## GraphiQL
-You can also experiment and run queries directly in the [GraphiQL interface](https://metadata.cloud.getdbt.com/graphiql), which is convenient for exploration. On the right hand side, there is a document explorer where you can see all possible nodes and fields.  Below is an example of what a query looks like in GraphiQL.  Note that you must authenticate via bearer auth with your token.
+
+You can experiment and run queries directly in the [GraphiQL interface](https://metadata.YOUR_ACCESS_URL/graphiql), replacing `YOUR_ACCESS_URL` with the [appropriate Access URL](/docs/deploy/regions-ip-addresses) for your region and plan. On the right hand side, there is a document explorer where you can see all possible nodes and fields.  Below is an example of what a query looks like in GraphiQL.  Note that you must authenticate via bearer auth with your token.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/metadata-querying.md]: @saimaddali Does the GraphiQL work for other access URLs?

Comment (runleonarun) [reply website/docs/docs/dbt-cloud-apis/metadata-querying.md]: Update: it does!

File: website/docs/docs/collaborate/manage-access/set-up-bigquery-oauth.md [hunk 2]
```diff
@@ -39,11 +39,10 @@ Fill in the application details as follows:
...
 | **Application type** | Web application |
 | **Name** | dbt Cloud |
-| **Authorized Javascript origins** | https://cloud.getdbt.com |
-| **Authorized Redirect URIs** | https://cloud.getdbt.com/complete/bigquery |
+| **Authorized Javascript origins** | https://YOUR_ACCESS_URL |
+| **Authorized Redirect URIs** | https://YOUR_ACCESS_URL/complete/bigquery |
 
-If you're a dbt Cloud single tenant customer, you need to replace `cloud.getdbt.com` with the hostname of
-your dbt Cloud instance.
+Replace `YOUR_ACCESS_URL` with the [appropriate Access URL](/docs/deploy/regions-ip-addresses) for your region and plan.
```
Comment (runleonarun) [inline website/docs/docs/collaborate/manage-access/set-up-bigquery-oauth.md]:
```suggestion
```

File: website/docs/docs/dbt-cloud-apis/metadata-querying.md [hunk 3]
```diff
@@ -6,33 +6,31 @@ Accounts on the _Team_ and _Multi-Tenant Enterprise_ plans can query the dbt Met
...
-```
-curl 'https://metadata.cloud.getdbt.com/graphql' \
-  -H 'authorization: Bearer <your token>' \
-  -H ‘content-type: application/json’
-  -X POST
-  --data <query>
-```
+  ```shell
+  curl 'https://metadata.YOUR_ACCESS_URL/graphql' \
+    -H 'authorization: Bearer <your token>' \
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/metadata-querying.md]:
```suggestion
    -H 'authorization: Bearer YOUR_TOKEN' \
```

File: website/docs/docs/dbt-cloud-apis/metadata-querying.md [hunk 4]
```diff
@@ -6,33 +6,31 @@ Accounts on the _Team_ and _Multi-Tenant Enterprise_ plans can query the dbt Met
...
+Currently, authorization of requests takes place [using a service token](/docs/dbt-cloud-apis/service-tokens). dbt Cloud admin users can generate a Metadata Only service token that is authorized to execute a specific query against the Metadata API.
 
 Once you've created a token, you can use it in the Authorization header of requests to the dbt Cloud Metadata API. Be sure to include the Token prefix in the Authorization header, or the request will fail with a `401 Unauthorized` error. Note that `Bearer` can be used in place of `Token` in the Authorization header. Both syntaxes are equivalent.
 
 ## Running Queries
 
-You can run queries by sending a `POST` request to the https://metadata.cloud.getdbt.com/graphql endpoint. Be sure to replace your token in the Authorization header with your actual API token.
+You can run queries by sending a `POST` request to the `https://metadata.YOUR_ACCESS_URL/graphql` endpoint, making sure to replace:
+* `YOUR_ACCESS_URL` with the [appropriate Access URL](/docs/deploy/regions-ip-addresses) for your region and plan.
+* Your token in the Authorization header with your actual API token.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/metadata-querying.md]:
```suggestion
* `YOUR_TOKEN` in the Authorization header with your actual API token.
```

File: website/docs/docs/dbt-cloud-apis/metadata-querying.md [hunk 5]
```diff
@@ -6,33 +6,31 @@ Accounts on the _Team_ and _Multi-Tenant Enterprise_ plans can query the dbt Met
...
+* QUERY_BODY with a JSON string, for example `{ "query": "<query text>" }`
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/metadata-querying.md]:
```suggestion
* `QUERY_BODY` with a JSON string, for example `{ "query": "<query text>" }`
```

File: website/docs/docs/collaborate/manage-access/set-up-sso-okta.md [hunk 6]
```diff
@@ -77,10 +74,11 @@ dashes. For example, the _login slug_ for dbt Labs would be
 `dbt-labs`. Login slugs must be unique across all dbt Cloud accounts,
 so pick a slug that uniquely identifies your company.
 
-On the **SAML Settings** page, enter the following values:
+On the **SAML Settings** page, enter the following values, replacing `YOUR_ACCESS_URL` with the [appropriate Access URL](/docs/deploy/regions-ip-addresses) for your region and plan.
+:
```
Comment (runleonarun) [inline website/docs/docs/collaborate/manage-access/set-up-sso-okta.md]:
```suggestion
```

File: website/docs/docs/collaborate/manage-access/set-up-sso-okta.md [hunk 7]
```diff
@@ -77,10 +74,11 @@ dashes. For example, the _login slug_ for dbt Labs would be
 `dbt-labs`. Login slugs must be unique across all dbt Cloud accounts,
 so pick a slug that uniquely identifies your company.
 
-On the **SAML Settings** page, enter the following values:
+On the **SAML Settings** page, enter the following values, replacing `YOUR_ACCESS_URL` with the [appropriate Access URL](/docs/deploy/regions-ip-addresses) for your region and plan.
```
Comment (runleonarun) [inline website/docs/docs/collaborate/manage-access/set-up-sso-okta.md]:
```suggestion
On the **SAML Settings** page, enter the following values, replacing `YOUR_ACCESS_URL` with the [appropriate Access URL](/docs/deploy/regions-ip-addresses) for your region and plan:
```

File: website/docs/docs/collaborate/manage-access/set-up-snowflake-oauth.md [hunk 8]
```diff
@@ -16,15 +16,15 @@ To enable Snowflake OAuth, you will need to create a [security integration](http
 
 ### Create a security integration
 
-In Snowflake, execute a query to create a security integration. Please find the complete documentation on creating a security integration for custom clients [here](https://docs.snowflake.net/manuals/sql-reference/sql/create-security-integration.html#syntax). You can find a sample `create or replace security integration` query below.
+In Snowflake, execute a query to create a security integration. Please find the complete documentation on creating a security integration for custom clients [here](https://docs.snowflake.net/manuals/sql-reference/sql/create-security-integration.html#syntax). In the following example `create or replace security integration` query, replace `YOUR_ACCESS_URL` with the [appropriate Access URL](/docs/deploy/regions-ip-addresses) for your region and plan. .
```
Comment (matthewshaver) [inline website/docs/docs/collaborate/manage-access/set-up-snowflake-oauth.md]:
```suggestion
In Snowflake, execute a query to create a security integration. Please find the complete documentation on creating a security integration for custom clients [here](https://docs.snowflake.net/manuals/sql-reference/sql/create-security-integration.html#syntax). In the following example `create or replace security integration` query, replace `YOUR_ACCESS_URL` with the [appropriate Access URL](/docs/deploy/regions-ip-addresses) for your region and plan.
```

File: website/docs/faqs/Accounts/transfer-account.md [hunk 9]
```diff
@@ -10,10 +10,10 @@ You can transfer your dbt Cloud [access control](/docs/collaborate/manage-access
...
 | **Team** | Existing account admins with account access can add users to, or remove users from the owner group. | 
 | **Enterprise** | Account admins can add users to, or remove users from a group with Account Admin permissions. | 
-| **If all account owners left the company** | If the account owner has left your organization, you will need to work with _your_ IT department to have incoming emails forwarded to the new account owner. Once your IT department has redirected the emails, you can request to reset the user password. Once you log in - you can change the email on the [Profile page](https://cloud.getdbt.com/settings/profile). |
+| **If all account owners left the company** | If the account owner has left your organization, you will need to work with _your_ IT department to have incoming emails forwarded to the new account owner. Once your IT department has redirected the emails, you can request to reset the user password. Once you log in - you can change the email on the Profile page when you replace `YOUR_ACCESS_URL` with the [appropriate Access URL](/docs/deploy/regions-ip-addresses) for your region and plan:  `https://YOUR_ACCESS_URL/settings/profile`. |
```
Comment (matthewshaver) [inline website/docs/faqs/Accounts/transfer-account.md]:
```suggestion
| **If all account owners left the company** | If the account owner has left your organization, you will need to work with _your_ IT department to have incoming emails forwarded to the new account owner. Once your IT department has redirected the emails, you can request to reset the user password. Once you log in, you can change the email on the Profile page when you replace `YOUR_ACCESS_URL` with the [appropriate Access URL](/docs/deploy/regions-ip-addresses) for your region and plan:  `https://YOUR_ACCESS_URL/settings/profile`. |
```

File: website/docs/faqs/API/rotate-token.md [hunk 10]
```diff
@@ -9,16 +9,19 @@ For security reasons and best practices, you should aim to rotate API keys every
 
 ### Steps to rotate API keys
 
-1. To rotate your [User API token](/docs/dbt-cloud-apis/user-tokens), send the following request: 
+1. To rotate your [User API token](/docs/dbt-cloud-apis/user-tokens), send the following request,  replacing `YOUR_USER_ID` , `YOUR_CURRENT_TOKEN`, and `YOUR_ACCESS_URL` with your information:
```
Comment (matthewshaver) [inline website/docs/faqs/API/rotate-token.md]:
```suggestion
1. To rotate your [User API token](/docs/dbt-cloud-apis/user-tokens), send the following request, replacing `YOUR_USER_ID`, `YOUR_CURRENT_TOKEN`, and `YOUR_ACCESS_URL` with your information:
```

**Review by matthewshaver — APPROVED** [review]

**Pull request merged**